instrumentation/README.md

```diff
@@ -70,4 +98,4 @@ Additionally the following guidelines for package composition need to be followe
   Also, packages MUST use the default `TracerProvider`, `MeterProvider`, and `Propagators` supplied by the `global` package if no optional one is provided.
 - All instrumentation packages MUST NOT provide an option to accept a `Tracer` or `Meter`.
 - All instrumentation packages MUST create any used `Tracer` or `Meter` with a name matching the instrumentation package name.
-- All instrumentation packages MUST create any used `Tracer` or `Meter` with a semantic version corresponding to the the version of this repository.
+- All instrumentation packages MUST create any used `Tracer` or `Meter` with a semantic version corresponding to the version of this repository.
```

Comment:
```suggestion
- All instrumentation packages MUST create any used `Tracer` or `Meter` with a semantic version corresponding to the version of the module containing the instrumentation.
```

As we start to stabilize some modules it will make less sense to talk about a single version of this repository.  Each module should probably have its own `version.go` file with a constant that can be managed by the release tooling.